taxonomies/categories.txt

```diff
@@ -32815,39 +32812,53 @@ hu:Ivójoghurtok, Iható joghurtok
 nl:Drinkyoghurts
 
 <en:Drinkable yogurts
-fr:Yaourts à boire goût fraise, Yaourts à boire fraise, Yaourts à boire saveur fraise
+fr:Yaourts à boire goût vanille, Yaourts à boire vanille, Yaourts à boire saveur vanille
+ciqual_food_code:en:19508
 
 <en:Drinkable yogurts
-fr:Yaourts à boire goût vanille, Yaourts à boire vanille, Yaourts à boire saveur vanille
+fr:Yaourts à boire goût framboise, yaourts-a-boire-gout-framboise
```

Comment:
All the strings like yaourts-a-boire-gout-framboise need to be removed if there are already existing corresponding synonyms that are correctly accented / capitalized.

Reply:
OK, done for yaourts-a-boire